src/y-webrtc.js

```diff
@@ -666,4 +696,10 @@ export class WebrtcProvider extends Observable {
     })
     super.destroy()
   }
+
+  addSignalingConn(url, conn) {
```

Comment:
`conn` is technically a function. Maybe we need to call it `createT` or something similar?